Fix DIM API loading now that we don't default accounts (#8831)
DestinyItemManager/DIM: docs/CHANGELOG.md

## Next

> * Fixed an issue where DIM Sync data (loadouts, tags, etc) could not show up until 10 minutes after loading DIM.

Reviewer: ```suggestion
* Fixed an issue where DIM Sync data (loadouts, tags, etc) could appear missing up for up to 10 minutes after loading DIM.
```
